index.js

```diff
@@ -233,27 +354,22 @@ const voicemeeter = {
 	}
 }
 
-// Create setter function
-const stripParametersNames = ["mono", "solo", "mute", "gain", "gate", "comp"];
-const busesParametersNames = ["mono", "mute", "gain"];
+const busesParametersNames = ["Mono", "Mute", "Gain"];
+const stripParametersNames = ["Mono", "Mute", "Solo", "MC", "Gain", "Pan_x", "Pan_y", "Color_x", "Color_y", "fx_x", "fx_y", "Audibility", "Gate", "Comp", "A1", "A2", "A3", "A4", "A5", "B1", "B2", "B3"];
```

Comment:
What about smaller version of Voicemeeter Potato, because they don't have A3, A4, or A5 ?

Reply:
It will just throw an error ?